tests/unit/test_contrib/test_pydantic/test_beanie_integration.py

```diff
@@ -1,13 +1,15 @@
-from typing import Optional
+from typing import TYPE_CHECKING, Optional, Type
 
 import beanie
-import pydantic
+
+if TYPE_CHECKING:
+    from pydantic import BaseModel
 
 from litestar.contrib.pydantic import PydanticDTO
 
 
-def test_generate_field_definitions_from_beanie_models() -> None:
-    class Category(pydantic.BaseModel):
+def test_generate_field_definitions_from_beanie_models(base_model: "Type[BaseModel]") -> None:
+    class Category(base_model):  # type: ignore[valid-type, misc]
```

Comment:
What about ignoring these in pyproject for the `test_pydantic` module?

Reply:
If it was just the `valid-type`, but `misc` is really broad, so I'd rather not